packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -260,7 +414,18 @@ class SliverGrid extends SliverMultiBoxAdaptorWidget {
   }
 }
 
+/// A sliver that contains a multiple box children that each fill the viewport.
+///
+/// [SliverFill] places its children in a linear array along the main axis. Each
+/// child is sized to fill the viewport, both in the main and cross axis.
+///
+/// See also:
+///
+///  * [SliverFixedExtentList], which has a configurable [itemExtent].
+///  * [SliverList], which does not require its children to have the same
+///    extent in the main axis.
```

Comment:
yeah that's the plan. i was hoping you'd point out what the useful parts were so i wouldn't have to figure it out myself :-)

i've been away from it for a few days though so maybe i'm refreshed enough that i can just do it